jobs/kubernetes/sig-testing/canaries.yaml

```diff
@@ -0,0 +1,24 @@
+periodics:
+- name: ci-kubernetes-e2e-prow-canary
+  interval: 1h
+  agent: kubernetes
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+      imagePullPolicy: Always
+      args:
+      - --bare
+      - --timeout=85
+      - --scenario=kubernetes-e2e
+      - --
+      - --check-leaked-resources
+      - --cluster=canary-e2e-prow
+      - --extract=ci/latest
+      - --gcp-zone=us-central1-f
+      - --ginkgo-parallel=25
+      - --provider=gce
+      - --test_args=--ginkgo.skip=\\[Slow\\]|\\[Serial\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\] --minStartupPods=8
+      - --timeout=65m
```

Comment:
nit: newlines missing at the end of these

Reply:
done